test/com/collections/MethodsTest.java

```diff
@@ -139,31 +140,52 @@ void canBePalindrome() {
         assertFalse(methods.canBePalindrome("kajakk"));
     }
 
-    @Test
-    void sumTwoIntArrays() {
+    @Test(dataProvider = "sumOfArrays1DataProvider")
```

Comment:
Why you need 2 data providers? What's the difference between those 2 tests?

Reply:
A ok, I see, you have two different sumOfArrayMethods. But still same data provider should be used here. Please name these methods and related tests in more appropriate way.